x/leverage/keeper/keeper_test.go

```diff
@@ -1197,6 +1197,27 @@ func (s *IntegrationTestSuite) TestReserveAmountInvariant() {
 	s.Require().False(broken)
 }
 
+func (s *IntegrationTestSuite) TestCollateralAmountInvariant() {
+	lenderAddr, _ := s.initBorrowScenario()
+
+	// The "lender" user from the init scenario is being used because it
+	// already has 1k u/umee for collateral
+
+	// it should not report any invariant
```

Comment:
These comments can be simplified to `// check invariant`

Reply:
done at https://github.com/umee-network/umee/pull/392/commits/89785e1cfeb64eda184b6823d84f2d435f1a9da8